src/commcare_cloud/ansible/roles/backups/templates/check_s3_backup.py.j2

```diff
@@ -3,24 +3,67 @@ import argparse
 import boto3
 import datetime
 
-#input environement and service from users
-
 parser = argparse.ArgumentParser(description="S3 Backup Status.")
-parser.add_argument('env', choices=['pna','softlayer','production','staging'] ,help='Environment Name')
 parser.add_argument('service', choices=['couch','blobdb','postgres'] ,help='Service Name')
 args = parser.parse_args()
 
-BUCKET_NAME = 'dimagi-'+args.env+'-'+args.service+'-backups'
+BUCKET_MAP = {
+  'couch': {{ couchdb_snapshot_bucket|tojson }},
+  'postgres': {{ postgres_snapshot_bucket|tojson }},
+  'blobdb': {{ blobdb_snapshot_bucket|tojson }},
+}
+
+PREFIX_MAP = {
+  'couch': "couchdb",
+  'postgres': "postgres_{{ deploy_env }}",
+  'blobdb': "blobdb",
+}
+
+BUCKET_NAME = BUCKET_MAP[args.service]
+PREFIX = PREFIX_MAP[args.service]
+
+endpoint = {{ aws_endpoint|tojson }}
+
+if not endpoint.startswith("http"):
```

Comment:
do you expect this script to work with http or only with https?

Reply:
It should work with either. It's just that "https://..." also starts with "http"